src/app-layout/__tests__/utils.tsx

```diff
@@ -87,7 +87,7 @@ export function describeEachAppLayout(callback: () => void) {
           (useMobile as jest.Mock).mockReset();
           (useVisualRefresh as jest.Mock).mockReset();
         });
-        callback();
+        callback(size as 'desktop' | 'mobile');
```

Comment:
minor: Instead of casting the value of the variable (technically not safe), it would be better to cast the array that you iterate over (`const size of ['desktop', 'mobile'] as const`)